samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,154 @@
+.. _direction_finding_connectionless_tx:
+
+Bluetooth: Direction finding connectionless beacon
+##################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The direction finding connectionless beacon sample demonstrates Bluetooth LE Direction Finding transmission.
+It uses the Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.
+
+The sample supports two Direction Finding modes:
+
+* Angle of Arrival
+* Angle of Departure
+
+By default, both modes are available in the sample.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix when operating in Angle of Departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Configuration
+*************
+
+|config|
+
+Angle of Arrival mode
+=====================
+
+To build this sample with Angle of Arrival mode only, set :makevar:`OVERLAY_CONFIG`
+to :file:`overlay-aoa.conf`.
```

Comment:
```suggestion
To build this sample with Angle of Arrival mode only, set ``OVERLAY_CONFIG`` to :file:`overlay-aoa.conf`.
```

Reply:
corrected line break and removed `:makevar:`